pkg/bindings/connection.go

```diff
@@ -13,12 +13,11 @@ import (
 	"strings"
 	"time"
 
+	sshCommon "github.com/containers/common/pkg/ssh"
```

Comment:
Why is sshCommon needed?  Not used elsewhere.

Reply:
fixed